cmd/machine-healthcheck/main.go

```diff
@@ -16,6 +16,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 )
 
+const defaultMetricsAddress = ":8084"
```

Comment:
Should we put these in the API package as publicly exposed, so that they are all in one place and can be used by other repos

```
const (
  MachineControllerMetricsBindAddress = ":8081"
  MachineSetControllerMetricsBindAddress = ":8082"
  NodeLinkControllerMetricsBindAddress = ":8083"
  MachineHealthCheckControllerMetricsBindAddress = ":8084"
)
```

Reply:
I think it's worth adding into `metrics.go` :+1: